### PR TITLE
feat: Tooltip for Assessment Creation page

### PR DESCRIPTION
**Pull request by Nimasheth**
[Tooltip for Assessment Creation page](https://www.notion.so/uwblueprintexecs/Tooltip-for-Assessment-Creation-page-abc04b696f0340b6a7b29f6c2a5baa15)
Screenshot 2023-03-27 at 5 42 52 PM
Screenshot 2023-03-27 at 5 42 54 PM
Screenshot 2023-03-27 at 5 42 56 PM

Adds tooltips for question types giving more information about each question type

## Checklist
- [X] My PR name is descriptive and in imperative tense
- [X] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [X] I have run the appropriate linter(s)
- [X] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/types/QuestionTypes.ts [hunk 1]
```diff
@@ -4,6 +4,12 @@ export enum QuestionType {
   MULTI_SELECT = "Multi-select",
 }
 
+export enum QuestionToolTips {
+  MULTIPLE_CHOICE = "Users will have to select the best response from a list of options",
+  MULTI_SELECT = "Users will have to select the correct responses from a list of options",
+  SHORT_ANSWER = "Users will have to type in their answers",
+}
+
```
Comment (joyce-shi) [inline frontend/src/types/QuestionTypes.ts:12]: i don't think you need an enum here, instead you can just store the text in `StudentAssessmentConstants`

File: frontend/src/constants/StudentAssessmentConstants.ts [hunk 2]
```diff
@@ -16,6 +16,14 @@ export const assessmentMetadata = {
...
 
+export const assessmentToolTips = {
+  questionTooltips: [
+    QuestionToolTips.MULTIPLE_CHOICE,
+    QuestionToolTips.MULTI_SELECT,
+    QuestionToolTips.SHORT_ANSWER,
+  ],
+};
+
 export const typeToImageMetadata = {
```
Comment (joyce-shi) [inline frontend/src/constants/StudentAssessmentConstants.ts]: you can just update this to `responseTypeMetadata` and add a `tooltip` field, i.e `tooltip: "Users will have to select the best response from a list of options"` under **Multiple Choice**.

then, you can delete `asssessmentToolTip` in constants, `QuestionToolTips` in question types and don't really need to update  the parameters in `StudentAssessment` and `QuestionTypeImages`

**Review by joyce-shi — CHANGES_REQUESTED** [review]

File: frontend/src/constants/StudentAssessmentConstants.ts (hunk 2 above)
Comment (Nimasheth) [reply frontend/src/constants/StudentAssessmentConstants.ts]: ok!

**Review by joyce-shi — APPROVED** [review]
LGTM!

**Pull request merged**